TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Service/ImportExport/NodeImportService.php

```diff
@@ -106,6 +107,12 @@ class NodeImportService
         'dimensionsHash' => array(),
         'dimensionValues' => array(),
         'properties' => array(),
+        'hiddenBeforeDateTime' => array(
```

Comment:
Just a nitpick / question: Should we use the short array syntax for new properties?